<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oneClass</class>
 <widget class="QMainWindow" name="oneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>one</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
color: rgb(255, 255, 255);
background-image: url(:/new/picture/beauty2.jpg);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_inp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>导入图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deb_8">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>369</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 181, 181);
font: 9pt &quot;黑体&quot;;
border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deb_4">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>209</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 181, 181);
font: 9pt &quot;黑体&quot;;
border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cla_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>中值滤波CPU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cla_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>经典核回归GPU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cla_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>中值滤波GPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deb_3">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>169</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 181, 181);
font: 9pt &quot;黑体&quot;;
background-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deb_6">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>289</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 181, 181);
font: 9pt &quot;黑体&quot;;
border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cla_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>自适应核回归GPU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cla_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>自适应核回归CPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deb_5">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>249</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 181, 181);
font: 9pt &quot;黑体&quot;;
background-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cla_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>均值滤波CPU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cla_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>均值滤波GPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deb_2">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>129</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 181, 181);
background-color: rgb(0, 0, 0);
font: 9pt &quot;黑体&quot;;
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deb_7">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>329</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 181, 181);
font: 9pt &quot;黑体&quot;;
border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cla">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>经典核回归CPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deb">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>89</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(181, 181, 181);
background-color: rgb(0, 0, 0);
font: 9pt &quot;黑体&quot;;
border-bottom-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lbl_deb_8</zorder>
   <zorder>lbl_deb_4</zorder>
   <zorder>btn_cla_7</zorder>
   <zorder>btn_cla_2</zorder>
   <zorder>btn_cla_8</zorder>
   <zorder>lbl_deb_3</zorder>
   <zorder>lbl_deb_6</zorder>
   <zorder>btn_cla_4</zorder>
   <zorder>btn_cla_3</zorder>
   <zorder>lbl_deb_5</zorder>
   <zorder>btn_cla_5</zorder>
   <zorder>btn_cla_6</zorder>
   <zorder>lbl_deb_2</zorder>
   <zorder>lbl_deb_7</zorder>
   <zorder>lbl_deb</zorder>
   <zorder>btn_inp</zorder>
   <zorder>btn_cla</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="one.qrc"/>
 </resources>
 <connections/>
</ui>
